CommunityFridgeMapApi/tests/unit/test_post_fridge_handler.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+
+import pytest
+
+from CommunityFridgeMapApi.functions.fridges.v1.app import FridgeHandler
+import json
+
+
+class DynamoDbMockPutItem:
+    def __init__(self):
+        pass
+
+    def put_item(self, TableName=None, Item=None, ConditionExpression=None):
+        pass
+
+
+class PostFridgeHandlerTest(unittest.TestCase):
+    def test_lambda_handler_success(self):
+        event = {
+            "body": '{"name": "greenpointfridge", "location": {"address":"9 W. Elm St.", "geoLat": "40.730610", "geoLng": "-73.935242"}}',
+            "httpMethod": "POST",
+            "pathParameters": {},
+            "queryStringParameters": {}
+        }
+        response = FridgeHandler.lambda_handler(
```

Comment:
Since the `DynamoDbMockPutItem` class does nothing, is this test even doing anything?

Reply:
It's the happy case. This still goes through the data validation logic in `db.py`